### DISCUSSION TITLE
javascript insert record

### DISCUSSION THREAD

**Discussion by alanbar** [supabase/supabase#3350] [Questions] [answered]
const { data, error } = await supabase
  .from('cities')
  .insert([
    { name: 'The Shire', country_id: 554 }
  ])

If name is a required field, the above will not insert
How can I test for the error(s) and see the associated message?

**Comment by silentworks** [comment, marked as answer]
You have the error object that is returned so you can inspect it and see what comes back. Also when doing single inserts, just pass an object to the insert function instead of an array.
